tests/test_pydeseq2.py

```diff
@@ -367,3 +367,49 @@ def test_anndata_init(tol=0.02):
     ).max() < tol
     assert (abs(r_res.pvalue - res_df.pvalue) / r_res.pvalue).max() < tol
     assert (abs(r_res.padj - res_df.padj) / r_res.padj).max() < tol
+
+
+def test_vst(tol=0.02):
+    """
+    Test the output of VST compared with DESeq2.
+    """
+
+    # Load data
+    counts_df = load_example_data(
+        modality="raw_counts",
+        dataset="synthetic",
+        debug=False,
+    )
+
+    clinical_df = load_example_data(
+        modality="clinical",
+        dataset="synthetic",
+        debug=False,
+    )
+
+    # Load R data
```

Comment:
can you shortly explain how this data was generated? Are you providing a script for it? if not perhaps just the lines you are using to get it? Alternatively you could add a short README or txt file withing the data directory explaining how each of the data files were generated

Reply:
I updated the readmes in [7dc8917](https://github.com/owkin/PyDESeq2/pull/99/commits/7dc891728d4327829b77f84535b43b3fef66b413) to explain how the data was generated.